final-project/scripts/main.js

```diff
@@ -1,10 +1,110 @@
 import "../styles/main.scss";
 
-// testing that `const` and arrow functions transpile correctly
-const before = "Hello -world!";
-const after = before
-  .split("")
-  .filter(char => char !== "-")
-  .join("");
-console.log({ before });
-console.log({ after });
+const url = "https://wt-4662f45b9eefda7172b747b28d23efdb-0.sandbox.auth0-extend.com/blog";
+var articlesIndex = 1;
```

Comment:
To avoid having a mess with global variables you could have separated the carousel into his own class